app/src/main/java/com/android/unio/model/event/Event.kt

```diff
@@ -54,7 +55,8 @@ data class Event(
     val location: Location = Location(),
     val types: List<EventType> = mutableListOf(EventType.OTHER),
     val maxNumberOfPlaces: Int = -1,
-    val numberOfSaved: Int = 0
+    val numberOfSaved: Int = 0,
+    val eventPictures: ReferenceList<EventUserPicture>,
```

Comment:
I'd say this is not a priority and could be added later on since everything is well enough separated. So in my opinion we should just stick to polishing this feature for the milestone.